src/yang/ietf-traffic-policy.yang

```diff
@@ -191,7 +178,15 @@ module ietf-traffic-policy {
           description
             "classifier entry name";
         }
-        uses inline-attr;
+        choice classifier-type {
```

Comment:
@aseemavi , I know I suggested that we make it a choice statement, but I am wondering if we are expecting more 'classifier-type' to be added in the future. Otherwise, I do not think we need to even make it a choice. Just have a container for 'inline', which can be a presence container, where if someone defines any of the inline-attr, it is understood that it is a inline classifier. Something like this:

```
container inline {
  presence "This container is present for when the device is configured for inline classification";
  uses inline-attr;
  description
    "Container for when an inline classification is desired.";
}
```

Reply:
make sense